### PR TITLE
add wiki link to 'platinum'

### PR DESCRIPTION
**Pull request by epogrebnyak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## The ReScience Journal
 
-[Re**Science**](https://rescience.github.io) is a *platinum open-access*
+[Re**Science**](https://rescience.github.io) 
+is a [*platinum open-access*](https://en.m.wikipedia.org/wiki/Open_access#Diamond/platinum_OA)
```
Comment (rougier) [inline README.md:4]: Because it is now a link, we can save the italic I think. Can you remove it please ?

**Comment by epogrebnyak** [comment]
Removed the italics

File: README.md (hunk 1 above)
Comment (epogrebnyak) [reply README.md]: #628b3e1 removes the italics. I was hestant too we need it.

**Comment by rougier** [comment]
Thanks.

**Pull request merged**